.vscode/launch.json

```
@@ -0,0 +1,43 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        // If you are running the compliance API tests (case 20) with this, pass the `--compliance-api-port=8385` to
+        // Ginkgo.
+        {
+            "name": "Launch Controller",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceFolder}/main.go",
+            "args": [
+                "--leader-elect=false",
+                "--log-level=4",
+                "-v=4",
+                "--enable-webhooks=false",
+                "--compliance-history-api-port=8385",
+                "--compliance-history-api-cert=dev-tls.crt",
+                "--compliance-history-api-key=dev-tls.key"
+            ],
+            "env": {
+                "WATCH_NAMESPACE": "",
+                "KUBECONFIG": "${workspaceFolder}/kubeconfig_hub",
+                "WATCH_NAMESPACE_COMPLIANCE_EVENTS_STORE": "open-cluster-management"
+            }
+        },
+        // Set FDescribe or FIt on the test to debug. Then set the desired breakpoint.
+        {
+            "name": "Launch Test Function (instructions in launch.json)",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceFolder}/test/e2e/e2e_suite_test.go",
+            "args": [
+                "-ginkgo.debug",
+                "-ginkgo.v"
+            ],
+            "env": {
+                "KUBECONFIG": "${workspaceFolder}/kubeconfig_managed_e2e"
```

Shouldn't this be a `hub` kubeconfig?
```suggestion
                "KUBECONFIG": "${workspaceFolder}/kubeconfig_hub_e2e"
```

---

Good catch